youtube_dl/extractor/polskieradio.py

```diff
@@ -15,6 +16,63 @@
 )
 
 
+class PolskieRadioProgrammeIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?polskieradio\.pl/(?P<id>\d+/\d+)(,.*)?(\?.*)?$'
```

Comment:
That'd be too ambiguous as it would match the URLs that are to be handled by the other extractor.